helm/gatekeeper-app/values.yaml

```diff
@@ -17,3 +17,6 @@ resources:
   requests:
     cpu: 100m
     memory: 256Mi
+project:
+  branch: "[[ .Branch ]]"
+  commit: "[[ .SHA ]]"
```

Comment:
Good to add these but architect-orb is 0.6.0 which might miss support for these.
https://github.com/giantswarm/gatekeeper-app/blob/979eee960e0aa13de2d1f965ca46d57f51dcb990/.circleci/config.yml#L3

Can you bump it to 0.8.9 which is the latest?

Reply:
yes